compiler/circle-interpreter-cffi-test/test.lst

```diff
@@ -0,0 +1,17 @@
+eval(Add_000)
```

Comment:
How did you select this list of Ops?

Reply:
It is from `record-minmax-conversion-test` as I think it is enough for checking the cffi features and without increasing the test time as much.